### DISCUSSION TITLE
Problems about the submission system

### DISCUSSION THREAD

**Discussion by a4rcvv** [yasserfarouk/scml#106] [General]
I submitted AgentNeko to the OneShot track, and I submitted my report today.

I found following problems about the submission system.

- After I submitted my report, my active agent AgentNeko disappeared from "Running in the official competition" field in the OneShot track leaderboard, and appeared in "Running in the official competition" field in the Standard and Collusion track leaderboard.
  - It seems that AgentSAS also has the same problem (#105).
- In relation to the above problem, I found that "For OneShot track" is unchecked, and "Run in collusion track" and "Run in standard track" are checked in the Agent Information. This is unintended behavior. I think it also happened after I submitted my report.
- Also, my *inactive* agents (Agent20123, Agent20135, AgentAMLegacy) is still on the "Running in the official competition" field in the OneShot track leaderboard.

![スクリーンショット 2022-06-15 20 58 55](https://user-images.githubusercontent.com/35623957/173822434-fae10679-0008-4d74-a71a-e0ef11221d4c.png)

My expecting behavior is the followings.

- AgentNeko will run in the OneShot track of the official competition.
- AgentNeko *won't* run in the Standard and Collusion track of the official competition.
- Agent20123, Agent20135, AgentAMLegacy *won't* run in any track of the official competition.

**Comment by a4rcvv** [comment]
@yasserfarouk I'd like to update about the issue.

- I confirmed that the bugs about AgentNeko is now resolved. Thank you.
- Agent20123, Agent20135, and AgentAMLegacy are still indicated to run in the official competition of OneShot track. Since I can't change my active agent, I can't access the Agent Information of them, and I can't uncheck their checkboxes.
  - Please note that they were submitted from the single account.

![スクリーンショット 2022-06-16 13 20 09](https://user-images.githubusercontent.com/35623957/173990462-429deb01-a230-4c1f-9015-607e41d4412f.png)

**Reply by yasserfarouk** [reply]
Thank you for the confirmation. Only AgentNeko will run in the official competition as it is now the ACTIVE agent associated with the report and your account. 

We will update the checkboxes of the other three to clarify that.

**Reply by a4rcvv** [reply]
I understood. Thank you.
